app/src/main/java/com/codesoom/assignment/application/TaskService.java

```diff
@@ -0,0 +1,47 @@
+package com.codesoom.assignment.application;
+
+import com.codesoom.assignment.TaskNotFoundException;
+import com.codesoom.assignment.domain.Task;
+import com.codesoom.assignment.domain.TaskRepository;
+import org.springframework.stereotype.Service;
+
+
+import java.util.List;
+@Service
```

Comment:
인텔리제이에서 `command + shift + L` 을 누르면 이런 import를 한번에 정리해 줄 거에요.
```suggestion
import com.codesoom.assignment.TaskNotFoundException;
import com.codesoom.assignment.domain.Task;
import com.codesoom.assignment.domain.TaskRepository;
import org.springframework.stereotype.Service;
import java.util.List;

@Service
```